src/components/Scorecard.vue

```diff
@@ -0,0 +1,430 @@
+<template>
```

Comment:
Does not match the mobile mockup in terms of spacing + the shade of green is different
<img width="539" alt="Screenshot 2022-04-13 at 12 07 35 PM" src="https://user-images.githubusercontent.com/13185699/163115261-43924a77-e956-49e8-a442-bf5d8acd6f6a.png">
<img width="451" alt="Screenshot 2022-04-13 at 12 07 52 PM" src="https://user-images.githubusercontent.com/13185699/163115280-528467aa-403b-4561-8a08-3ae5431ec329.png">

Reply:
I talked to Karn about the green, he said it's fine until I keep the same shade of green everywhere.